website/docs/d/pi_cloud_connection.html.markdown

```diff
@@ -37,28 +32,26 @@ data "ibm_pi_cloud_connection" "example" {
   ```
 
 ## Argument reference
-
 Review the argument references that you can specify for your data source.
 
 - `pi_cloud_instance_id` - (Required, String) The GUID of the service instance associated with an account.
 - `pi_cloud_connection_name` - (Required, String) The cloud connection name to be used.
 
 ## Attribute reference
-
 In addition to all argument reference list, you can access the following attribute references after your data source is created.
 
-- `id` - (String) The unique identifier of the cloud connection.
-- `classic_enabled` - (Bool) Is classic endpoint destination enabled?
+- `classic_enabled` - (Bool) Enable classic endpoint destination.
```

Comment:
Change to (Boolean) to match other parts of the IBM terraform provider documentation.